apps/client/src/locales/messages.de.xlf

```diff
@@ -7258,6 +7258,14 @@
           <context context-type="linenumber">280</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="b225488f8b209e9704760dc9f5d99845a5d07bf6" datatype="html">
+        <source>Oops! Could not find any assets.</source>
+        <target state="translated">Hoppla! Es konnten keine Assets gefunden werden.</target>
```

Comment:
In the whole app we are using _Ups!_, can you change for consistency reasons? 😃

Reply:
Oh, yeah of course! Sorry for that oversight.